packages/cozy-stack-client/src/DocumentCollection.spec.js

```diff
@@ -329,6 +329,24 @@ describe('DocumentCollection', () => {
       const resp = await collection.find({ done: false })
       expect(resp.data[0]).toHaveDocumentIdentity()
     })
+
+    it('should index the specified fields', async () => {
+      const collection = new DocumentCollection('io.cozy.todos', client)
+      await collection.find(
+        { done: { $exists: true } },
+        { indexedFields: ['label'], sort: [{ label: 'desc' }] }
+      )
+      expect(client.fetchJSON).toHaveBeenLastCalledWith(
+        'POST',
+        '/data/io.cozy.todos/_find',
```

Comment:
I don't think this test really tests that the proper fields are indexed. Shouldn't we check for a call to `'/data/io.cozy.todos/_index'`?

Reply:
Yes, actually both are necessary. The `_find` call to ensure the sort doesn't include the label, and the `_index` call to make sure the index is created.